sdk/cosmos/azure-cosmos/README.md

```diff
@@ -681,10 +681,30 @@ indexing_policy = {
         ],
         "vectorIndexes": [
             {"path": "/vector1", "type": "flat"},
-            {"path": "/vector2", "type": "quantizedFlat"}
+            {"path": "/vector2", "type": "quantizedFlat"},
+            {"path": "/vector2", "type": "diskANN"}
         ]
     }
 ```
+
+For vector index types of diskANN and quantizedFlat, there are additional options available as well. These are:
+
+quantizationByteSize - the number of bytes used in product quantization of the vectors. A larger value may result in better recall for vector searches at the expense of latency. This applies to index types diskANN and quantizedFlat. The allowed range is between 1 and 256, and the default value is 64.
+
+vectorIndexShardKey - the list of string containing the shard keys used for partitioning the vector indexes. This applies to index types diskANN and quantizedFlat.
+
+indexingSearchListSize - which represents the size of the candidate list of approximate neighbors stored while building the diskANN index as part of the optimization processes. The allowed range is between 25 and 500.
```

Comment:
Nit, but this is only available for `diskANN`, where the other two are available for both `quantizedFlat` and `diskANN`.  Not sure it's worth trying to separate more though.